test/invariant/fuzz/helpers/Actors.t.sol

```diff
@@ -44,6 +44,7 @@ contract Actors is Utils {
     ];
 
     mapping(address actor => address anchor) internal _ghost_anchorOf;
+    mapping(bytes32 profileId => address actor) internal _ghost_profileIdToActor;
```

Comment:
can't you use `Registry.profilesById(id).owner`?
imo, we should recompute/reuse as much as possible from the target contracts, it's so tricky to track ghost when they're everywhere;)

Reply:
good idea